fhir-server-test/src/test/java/com/ibm/fhir/server/test/ExportOperationTest.java

```diff
@@ -26,68 +31,54 @@
 import com.ibm.fhir.model.format.Format;
 import com.ibm.fhir.model.generator.FHIRGenerator;
 import com.ibm.fhir.model.generator.exception.FHIRGeneratorException;
+import com.ibm.fhir.model.resource.Condition;
+import com.ibm.fhir.model.resource.Group;
+import com.ibm.fhir.model.resource.Group.Member;
+import com.ibm.fhir.model.resource.Observation;
 import com.ibm.fhir.model.resource.Parameters;
 import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.model.resource.Patient;
+import com.ibm.fhir.model.test.TestUtil;
 import com.ibm.fhir.model.type.Instant;
+import com.ibm.fhir.model.type.Reference;
 
 /**
- * These tests exercise the $export operation, a BulkData specification defined operation.
- * 
- * @author pbastide
+ * These tests exercise the $export operation, a BulkData specification defined operation
  *
  */
 public class ExportOperationTest extends FHIRServerTestBase {
-
     public static final String TEST_GROUP_NAME = "export-operation";
-
     public static final String PATIENT_VALID_URL = "Patient/$export";
-    public static final String GROUP_VALID_URL = "Group/$export";
+    public static final String GROUP_VALID_URL = "Group/?/$export";
```

Comment:
? gets replaced?

Reply:
yes, ? will be replace with the group id